R/get-user-teams.R

```diff
@@ -0,0 +1,25 @@
+#' Get teams a user belongs to
+#'
+#' Looks up the teams a user belongs to. By default, it looks for teams of the
+#' current logged in user. You must be logged in to Synapse to use this
+#' function.
+#'
+#' @param user Synapser user object (e.g. output from
+#'   [synapser::synGetUserProfile()])
+#' @return Character vector of team IDs the user belongs to
+#' @examples
+#' \dontrun{
+#' get_user_teams()
+#' get_user_teams("karawoo")
```

Comment:
This example would fail.  You would need to pass in `synGetUserProfile("karawoo")`. Looks good otherwise!

Reply:
Good catch, I changed the behavior at one point and forgot to update the example. Fixed now. Thanks!